schemas/student.py

```diff
@@ -23,49 +22,47 @@ class CreateStudentIn(BaseModel):
     gender: str
 
     @validator('full_name')
-    def validate_full_name(value):
-        full_name = value.split()
-        if not full_name:
-            raise ValueError("Прізвище та ім'я студента обов'язкові до заповнення!")
-        elif len(full_name) < 2:
-            raise ValueError("Прізвище та ім'я студента обов'язкові до заповнення!")
+    def validate_full_name(value):  # TODO may be there is need to use staticmethod decorator
+        full_name = value.split()   # TODO use hint or so
```

Comment:
Sure, you can use type hints for (something like that):
```python
def validate_full_name(value: str):
    full_name: List[str, str] = value.split()
```

Reply:
not resolved but ok for now